src/lib/runtime.ts

```diff
@@ -175,7 +174,7 @@ export class BaseAPI {
 }
 
 async function isFormData(value: unknown): Promise<boolean> {
-  const FormData = await getFormDataCls();
+  const FormData = globalThis.FormData;
```

Comment:
Same here, just use global Blob and FormData

You can also remove this method and just use `overriddenInit.body instanceof FormData` where it's being used